src/curl.c

```diff
@@ -262,8 +262,20 @@ void
 curl_env_destroy(struct curl_env *env)
 {
 	assert(env);
-	if (env->multi != NULL)
+	if (env->multi) {
```

Comment:
Is it possible to avoid changing this line. Explicit comparison was better than implicit.

Reply:
Reverted:

```diff
--- a/src/curl.c
+++ b/src/curl.c
@@ -262,7 +262,7 @@ void
 curl_env_finish(struct curl_env *env)
 {
        assert(env);
-       if (env->multi) {
+       if (env->multi != NULL) {
                CURL **list = curl_multi_get_handles(env->multi);
                if (list) {
                        for (int i = 0; list[i]; i++) {
```